src/libinjector/injector.c

```diff
@@ -1265,23 +1275,65 @@ static void print_injection_info(output_format_t format, vmi_pid_t pid, uint64_t
 
     char* escaped_arguments = g_strescape(arguments, NULL);
 
-    switch (format)
+    if (INJECT_RESULT_SUCCESS == injector->result)
```

Comment:
Perhaps instead of this long if/elseif/else route do a `switch` on the `injector->result`